clippy.toml

```diff
@@ -3,4 +3,5 @@ disallowed-methods = [
   { path = "xtra::Address::do_send", reason = "discards the return value, possibly swallowing an error" },
   { path = "xtra::message_channel::MessageChannel::do_send", reason = "discards the return value, possibly swallowing an error" },
   { pant = "xtra::Context::notify_interval", reason = "does not wait for response from the previous handler, prefer `xtra_ext::Address::send_interval`" },
```

Comment:
There is a typo here which tells me that clippy did actually not read both of these files before!

Reply:
lol, indeed! and because it was a valid word my spellchecker didn't complain 😆